animationAssets.js

```diff
@@ -0,0 +1,214 @@
+// eslint-disable-next-line camelcase
+const path = require('path')
+const util = require("util")
+const fs = require("pn/fs") // https://www.npmjs.com/package/pn
+const fsNode = require("fs-extra") // https://www.npmjs.com/package/pn
+const xml = require("node-xml-lite")
+const svg2png = require("svg2png")
+const pathToAndroidAssetsAnimation = 'android/app/src/main/assets/animations'
```

Comment:
for drawable-xxxx to work images need to be in "res" folder not assets and flat without hierarchy
so call images res/animation_name.png, res/drawable-xxxx/animation_name.png
since lottie doesnt support this at the moment generate one hires png in assets like now